Source/AlchemyVisionV1/AlchemyVision.swift

```diff
@@ -38,14 +40,10 @@ public class AlchemyVision {
      Create an `AlchemyVision` object.
 
      - parameter apiKey: The API key credential to use when authenticating with the service.
-     - parameter serviceURL: The base URL to use when contacting the service.
      */
-    public init(
-        apiKey: String,
-        serviceURL: String = "http://gateway-a.watsonplatform.net/calls")
+    public init(apiKey: String)
```

Comment:
I didn't think about this before, but without the urls (and their long default values) in the init, we can make the style nicer here and include the opening bracket on the same line.